DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupDelegates.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model GroupDelegatesViewModel
+
+@{
+  ViewData["Title"] = "Group delegates";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/groupDelegates.css")" asp-append-version="true">
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="_DelegateGroupBreadcrumbs" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="_DelegateGroupsSideNavMenu" model="Model.NavViewModel" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Group delegates</h1>
+      </div>
+      <div class="nhsuk-grid-column-one-third all-delegates-button-group">
+        <a class="nhsuk-button all-delegates-button" role="button">
+          Add
+        </a>
+      </div>
+    </div>
+
+    <form method="get">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          @if (!Model.GroupDelegates.Any())
+          {
+            <p class="nhsuk-u-margin-top-4" role="alert">
+              <b>No delegates found.</b>
+            </p>
+          }
+          else
+          {
+            <div id="searchable-elements">

```

Comment:
It probably doesn't technically need it since we are only doing no-JS pagination here, but the pagination is tied to the search/sort/filter functionality, so I've kept the layout the same. If we were to implement JS pagination (this would require modifying the search/sort/filter to make them each optional) this would be required.

Reply:
It sounds to me like it's better to leave the id out if we doesn't need it (just tried it and it seems fine without). If we've got some docs on how to implement JS pagination (or a fairly clear example to follow) then the only benefit of having this already here is that it'll save half a minute of dev down the line. Contrast that with the negatives of
- making future devs think that these elements are searchable or use JS search/sort/filter
- having extra complexity that we don't need (so if we changed the id `searchable-elements`, say, we'd have to remember that it's used here)

But maybe I've mis-stated the benefits. What do you think?

Reply:
Yeah I agree with that assessment (unless I find that the styles break, in which case I should fix the styles so that they don't rely on this).